PlayList.java

```diff
@@ -123,8 +215,22 @@ public String titleOfShortestTrack() {
      *  rather than returning a new, sorted playlist, the method sorts
      *  the list on which it was called (this list). */
     public void sortedInPlace() {
-        // Uses the selection sort algorithm,  
-        // calling the minIndex method in each iteration.
-        //// replace this statement with your code
+
+        for(int i=0; i< size -1; i++)
+        {
+            int min=i;
+            for(int j = i+1 ; j< size; j++)
+            {
+                if(tracks[j].getDuration()< tracks[min].getDuration())
+                {
+                    min =j;
+                }
+            }
```

Comment:
You already implemented it, then call it.
```suggestion
            int min=minIndex(i);
```